### DISCUSSION TITLE
Issues after upgrading to zsh 5.8.1

### DISCUSSION THREAD

**Discussion by Pilgrim1379** [marlonrichert/zsh-autocomplete#415] [Q&A] [answered]
After upgrading zsh from 5.8_1 to 5.81 (homebrew), I've noticed a few issues with autocompletion.
1. There is a noticeable lag when I type each character at the prompt.
2. Autocomplete candidates just flash below the prompt when I press `<tab>` plus it doesn't auto list although I have `zstyle ':autocomplete:*' min-input 0`. By that I mean day I type `cd `, at this point I get no completion candidates listed. So I continue with `cd p`, still no completion candidates listed at this point. Then I hit `<tab>` and below the prompt I see the list of possible candidates just flash and disappear.

When I change zsh to 5.8 (default for macos) everything works great again.

**Comment by marlonrichert** [comment, marked as answer]
Duplicate of #416.
